index.html

```diff
@@ -227,6 +227,9 @@ <h2 id="x2.link-type-preload">Link type "<code>preload</code>"</h2>
         memory cache and be re-usable at least once within the lifetime of a
         fetch group.</p>
       </div>
+      <div class="note">
+        Link HTTP response header should be processed for all <a>request destination</a>s.
```

Comment:
https://github.com/w3c/respec/wiki/User's-Guide#definitions-and-linking, or.. "all types of request destination"?